app/consumer-democracy/app.go

```diff
@@ -137,7 +135,7 @@ var (
 		ccvdistr.AppModuleBasic{},
 		gov.NewAppModuleBasic(
 			// TODO: eventually remove upgrade proposal handler and cancel proposal handler
-			paramsclient.ProposalHandler, distrclient.ProposalHandler, upgradeclient.ProposalHandler, upgradeclient.CancelProposalHandler,
```

Comment:
Why this removal?

Reply:
as of 47 we no longer have a distribution client